CHANGES.md

```diff
@@ -1,5 +1,9 @@
 ## edge-20.1.3
 
+An update to the Helm charts has caused a **breaking change** for users who
+have installed Linkerd using Helm. In order to make the purpose of the
+`NoInitContainer` parameter more explicit, it has been renamed to `CniEnabled`.
+
 * CLI
   * Introduced `linkerd check --pre --linkerd-cni-enabled`, used when the CNI
```

Comment:
Might be that I am not native speaker but this sounds a bit strange to me. How about something like "Introduced CNI checks to confirm the CNI plugin is installed and ready".  On another note this check is ran not only in `--pre` stage but also in `linkerd check` **if the cni plugin is present**

Reply:
@zaharidichev?

```
* Introduced CNI checks to confirm the CNI plugin is installed and ready;
  this is done through `linkerd check --pre --linkerd-cni-enabled` before installation, and
  `linkerd check` if the CNI plugin is present
```

Reply:
That sounds correct.